LEAF_Request_Portal/admin/templates/mod_workflow.tpl

```diff
@@ -1881,66 +1845,65 @@
+                    jsPlumb.draggable('step_-1');
+                }
+                if (endPoints[0] == undefined) {
+                    endPoints[0] = jsPlumb.addEndpoint('step_0', {anchor: 'Continuous'}, endpointOptions);
+                    jsPlumb.draggable('step_0');
+                }
 
-                    // draw connector
-                    for (let i in res) {
-                        var loc = 0.5;
-                        switch (res[i].actionType) {
-                            case 'sendback':
-                                loc = 0.30;
-                                break;
-                            case 'approve':
-                                loc = 0.5;
-                                break;
-                            case 'concur':
-                                loc = 0.5;
-                                break;
-                            case 'defer':
-                                loc = 0.25;
-                                break;
-                            case 'disapprove':
-                                loc = 0.75;
-                                break;
-                            default:
-                                break;
-                        }
-                        if (res[i].nextStepID == 0 &&
-                            res[i].actionType == 'sendback') {
-                            jsPlumb.connect({
-                                    source: 'step_' + res[i].stepID,
-                                    target: 'step_-1',
-                                    paintStyle: {stroke: 'red'},
-                                    overlays: [
-                                        ["Label", {
-                                                id: 'stepLabel_' + res[i].stepID + '_0_' + res[i]
-                                                    .actionType,
-                                                cssClass: "workflowAction",
-                                                label: res[i].actionText,
-                                                location: loc,
-                                                parameters: {'stepID': res[i].stepID,
-                                                'nextStepID': 0,
-                                                'action': res[i].actionType
-                                            },
-                                            events: {
-                                                click: function(overlay, evt) {
-                                                    params = overlay.getParameters();
-                                                    showActionInfo(params, evt);
-                                                }
-                                            }
+                // draw connector
+                for (let i in res) {
+                    var loc = 0.5;
+                    switch (res[i].actionType) {
```

Comment:
tolower()  And looking lower in the code, you're going to want to make that change further up.  I'm seeing other comparisons.